assets/i18n/en.json

```diff
@@ -234,8 +234,10 @@
     "I18N_END_CHAPTER_MILESTONE_MESSAGE_3": "You just completed your 10th chapter!",
     "I18N_END_CHAPTER_MILESTONE_MESSAGE_4": "You just completed your 25th chapter!",
     "I18N_END_CHAPTER_MILESTONE_MESSAGE_5": "You just completed your 50th chapter!",
+    "I18N_END_CHAPTER_MILESTONE_PROGRESS_MESSAGE": "{chaptersToGo, plural, one{Complete 1 more chapter to reach your next milestone!} other{Complete # more chapters to reach your next milestone!}}",
```

Comment:
This seems a bit too complicated to put into a translate string. Can we make this simpler, probably by splitting "Complete 1 more chapter to reach your next milestone!" and "Complete # more chapters to reach your next milestone!" into two translate strings.

Reply:
This is actually [the recommended way](https://github.com/oppia/oppia/wiki/How-to-develop-for-i18n#plurals-and-gender) of handling texts which change based on the plurality of one of the objects in the sentence. There are other examples of this too, like [this](https://github.com/oppia/oppia/blob/8f04c4b559eab93f28c155606106e9e86501cb5f/assets/i18n/en.json#L557) and [this](https://github.com/oppia/oppia/blob/8f04c4b559eab93f28c155606106e9e86501cb5f/assets/i18n/en.json#L469), which have already have correct translations, so I don't think this string will cause any issues either.

Reply:
Interesting, okay.